src/sqs/queues.ts

```diff
@@ -68,20 +70,46 @@ const sqsQueues = {
 	),
 
 	start: () => {
+		logger.info("Starting queues");
 		sqsQueues.backfill.start();
 		sqsQueues.push.start();
 		sqsQueues.discovery.start();
 		sqsQueues.deployment.start();
 		sqsQueues.branch.start();
+		logger.info("All queues started");
 	},
 
-	stop: () => {
-		sqsQueues.backfill.stop();
-		sqsQueues.push.stop();
-		sqsQueues.discovery.stop();
-		sqsQueues.deployment.stop();
-		sqsQueues.branch.stop();
-	}
-}
+	stop: async () => {
+		logger.info("Stopping queues");
+		await Promise.all([
+			sqsQueues.backfill.stop(),
+			sqsQueues.push.stop(),
+			sqsQueues.discovery.stop(),
+			sqsQueues.deployment.stop(),
+			sqsQueues.branch.stop()
+		]);
+		logger.info("All queues stopped");
+	},
 
-export default sqsQueues
+	purge: async () => {
```

Comment:
As far as I can tell, the `purge()` function isn't used anywhere, is it? It's a dangerous function to have around!

Reply:
That's a very good point.  I had it originally for an `afterEach` for all SQS tests, but decided to be more granular.  I'll remove it.

Reply:
So the Purge function doesn't guarantee consistency. Messages can still be on the queue after we called it